packages/postgres-queries-builder/src/queries/select/SelectByOrderedIdentifiers.ts

```diff
@@ -33,7 +33,9 @@ ON ${table}.${primaryKey}::varchar=x.${primaryKey}
 ORDER BY x.ordering;`;
 
     const parameters = flatten(
-        sanitizeParameter([primaryKey], { [primaryKey]: values }),
+        sanitizeParameter([primaryKey], {
+            [Array.isArray(primaryKey) ? primaryKey[0] : primaryKey]: values,
```

Comment:
Are you sure that selecting first value of the array is the expected way of working ?

Reply:
Nope ... But: 
* tests don't fail,
* giving an array as an index seems like a bad idea...

Reply:
:+1: